lamprey.lic

```diff
@@ -0,0 +1,57 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#lamprey
+=end
+
+custom_require.call(%w[events common-travel common common-items])
+
+class Lamprey
+  include DRC
+  include DRCI
+  include DRCT
+
+  def initialize
+    walk_to_lamprey
+    loop do
+      play_lamprey
+      check_prize
+    end
+  end
+
+  def walk_to_lamprey
+    message("*** Heading to Lamprey! ***")
+    walk_to(16_173)
+  end
+
+  def play_lamprey
+    case bput('get lamprey from trough', 'You bite down', 'You just got a lamprey.')
+    when 'You just got a lamprey.'
+      message("*** You will need to wait 10 minutes to play again! ***")
+      exit
+    end
+  end
+
+  def check_prize
+    junk_list = get_settings.darkbox_junk.map { |x| /#{x}/i }
+    prize = DRC.right_hand
```

Comment:
💡 If you're relying on the prize to be in your right hand, should you empty hands before starting?